src/cljs/rems/administration/administration.cljs

```diff
@@ -19,7 +19,7 @@
      (text :t.administration/back)]))
 
 (defn navigator []
-  [:div#administration-menu.navbar.mb-4.mr-auto.ml-auto
+  [:div#administration-menu.navbar.mb-3.mr-auto.ml-auto
```

Comment:
This doesn't actually seem to change anything. Perhaps because the changed `h1` margins.

Reply:
that is the point. In cases where we have administration navbar and h1 below there is no too much space between them and in turn when h1 we have decent space on top.

long story short, it does not change anything in the places where admin navbar meets the h1

Reply:
All the pages should have the navbar and the `h1` at the top so we could just remove this change?

Reply:
not every page has administration navbar

Reply:
without administration navbar
<img width="1287" alt="Screenshot 2021-01-14 at 13 07 33" src="https://user-images.githubusercontent.com/7344718/104583144-789c5c00-5669-11eb-82c9-ad9ed484fea9.png">

with administration navbar
<img width="1303" alt="Screenshot 2021-01-14 at 13 07 51" src="https://user-images.githubusercontent.com/7344718/104583188-8356f100-5669-11eb-813d-baeddfcc12a2.png">

Reply:
But you are adding bottom margin to the admin navbar so surely you only need to consider the pages that have administration navbar (and all pages have `h1`)?

Reply:
ok, I have completely lost trail of your point

Reply:
the problem only occurs when navbar meets h1, there is no point of changing it on every page.

Reply:
I'm not sure what problem you are referring to here. What I'm thinking is that the margins of the `navbar` and `h1` collapse so there is not too much space.

Reply:
can we just agree that it looks ok and works with current UI and move on?

Reply:
I think I was saying that the old one also looked ok and worked and therefore the diff was unnecessary. We can leave this like it is now, the change is minor.